tests/test_encoding.py

```diff
@@ -0,0 +1,56 @@
+# coding=utf-8
+"""Encoding tests"""
+
+from glob import glob
+import logging
+import os
+import shutil
+import tempfile
+import unittest
+
+import pytest
+
+import fiona
+
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+# Uncomment when merged to master, where we're using pytest.
+# We're still holding our noses in maint-1.7.
```

Comment:
You're planning to cherry-pick this into maint-1.7?

Reply:
Not a cherry pick, but a separate PR.